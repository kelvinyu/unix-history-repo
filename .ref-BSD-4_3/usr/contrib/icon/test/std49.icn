record array(a,b,c,d,e,f,g)

procedure dummy(u,v,x,y,z)
   suspend u | v
   return x
end

procedure main()
   write("e := create 1 to 10 ----> ",image(e := create 1 to 10) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("e := ^e ----> ",image(e := ^e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("@e ----> ",image(@e) | "none")
   write("*e ----> ",image(*e) | "none")
   write("*e ----> ",image(*e) | "none")
   write("image(e) ----> ",image(image(e)) | "none")
   write("e || e ----> ",image(e || e) | "none")
end
