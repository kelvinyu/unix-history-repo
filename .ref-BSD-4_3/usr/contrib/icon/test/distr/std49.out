e := create 1 to 10 ----> co-expression(0)
@e ----> 1
@e ----> 2
@e ----> 3
e := ^e ----> co-expression(0)
@e ----> 1
@e ----> 2
@e ----> 3
@e ----> 4
@e ----> 5
@e ----> 6
@e ----> 7
@e ----> 8
@e ----> 9
@e ----> 10
@e ----> none
@e ----> none
@e ----> none
@e ----> none
*e ----> 10
*e ----> 10
image(e) ----> "co-expression(10)"
Run-time error 103 at line 31 in std49.icn
string expected
offending value: co-expression
